### DISCUSSION TITLE
I'm almost blind. Is there a tool to help me handle colors?

### DISCUSSION THREAD

**Discussion by senpro-ingwersenk** [saadeghi/daisyui#3195] [General]
Hello!

Basically, to make a long story short, my company has a logo and a defined corporate identity and I would like to incorporate those colors as part of a theme. But, no matter which "theme generator" or whatever I open up, I am stuck having to define several levels of a color scheme.

I am almost blind. I suck at this. A **lot**! xD However, I would really like the app I am developing to not look terrible, y'know?

As google isn't what it was once before and some gems are often harder to come by, are there good tools that can help me do that? 

Effectively, if you take a look at the company website https://senpro.de and look at the logo, you will see the colors I mean.

I have the color codes for red, green and two levels of grey. How can I turn that into a proper palette that I can then use with DaisyUI components?

My long-term plan is to use TailwindCSS + DaisyUI together with (perhaps, anyway) either HedlessUI or Radix Primitives. My "framework" of choice is Preact since I hope to profit off of it's size advantage over regular React.

Thank you, and kind regards!

**Comment by saadeghi** [comment]
daisyUI theme generator would be helpful:  
https://daisyui.com/theme-generator/

There's also a community version:  
https://themes.ionevolve.com/

I made an example theme here based on the colors of your site:  
https://play.tailwindcss.com/AUGLFXwt4T?file=config

I hope it helps.

Let me know if you have a question.
